packages/flutter_test/lib/src/matchers.dart

```diff
@@ -282,6 +282,8 @@ Matcher coversSameAreaAs(Path expectedPath, { @required Rect areaToCompare, int
 ///
 /// See also:
 ///
+///  * [Writing a golden file test](https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter),
```

Comment:
this doesn't explain how to create a golden file test, it explains specifically how to create a golden file test for people working on the `flutter` package itself.

Reply:
Okay, I've changed it to explain to use `flutter test --update-goldens`.